packages/react/select/src/Select.tsx

```diff
@@ -624,7 +625,25 @@ const SelectContentImpl = React.forwardRef<SelectContentImplElement, SelectConte
       }
     });
 
+    const handleItemSeen = React.useCallback(
+      (node: SelectItemElement | null, value: string, disabled: boolean) => {
+        const isFirstSeenValidItem = !firstItemRef.current && !disabled;
+        const isSelectedItem = context.value !== undefined && context.value === value;
```

Comment:
Is `context.value !== undefined` so that we can support empty strings as values for items?

Reply:
Yes exactly.